pkg/ingester/ingester.go

```diff
@@ -760,7 +769,7 @@ func (i *Ingester) PushWithCleanup(ctx context.Context, pushReq *push.Request) (
 
 		minAppendTime, minAppendTimeAvailable := db.Head().AppendableMinValidTime()
 
-		err = i.pushSamplesToAppender(userID, req.Timeseries, persistentApp, startAppend, &persistentStats, updateFirstPartial, activeSeries, i.limits.OutOfOrderTimeWindow(userID), minAppendTimeAvailable, minAppendTime, true)
+		err = i.pushSamplesToAppender(userID, req.Timeseries, persistentApp, startAppend, &persistentStats, updateFirstPartial, activeSeries, i.limits.OutOfOrderTimeWindow(userID), minAppendTimeAvailable, minAppendTime, true, false)
```

Comment:
[nit] I don't think weed to pass "exemplars support" bool anymore. We can infer it from `ephemeral bool`.

Reply:
Done in https://github.com/grafana/mimir/pull/4004/commits/91da861ff7c3264b52d1c765adc63e41be7ef98b